tgui/packages/tgui/interfaces/Clipboard.js

```diff
@@ -0,0 +1,117 @@
+import { useBackend } from "../backend";
+import {
+  Box,
+  Button,
+  Divider,
+  LabeledList,
+  Flex,
+  Section,
+} from "../components";
+import { Window } from "../layouts";
+
+export const Clipboard = (props, context) => {
+  const { act, data } = useBackend(context);
+  const { pen, top_paper, top_paper_ref, paper, paper_ref } = data;
+  return (
+    <Window title="Clipboard" width={400} height={500}>
+      <Window.Content backgroundColor="#704D25" scrollable>
+        <Section>
+          {pen ? (
+            <LabeledList>
+              <LabeledList.Item
+                label="Pen"
+                buttons={
+                  <Button icon="eject" onClick={() => act("remove_pen")} />
+                }
+              >
+                {pen}
+              </LabeledList.Item>
+            </LabeledList>
+          ) : (
+            <Box color="white" align="center">
+              No pen attached!
+            </Box>
+          )}
+        </Section>
+        <Divider />
+        {top_paper ? (
+          <>
+            <Flex
+              color="black"
+              backgroundColor="white"
+              style={{ padding: "2px 2px 0 2px" }}
+            >
+              <Flex.Item align="center" grow={1}>
+                <Box align="center">{top_paper}</Box>
+              </Flex.Item>
+              <Flex.Item>
+                <Button
+                  icon={pen ? "pen" : "eye"}
+                  onClick={() => act("edit_paper", { ref: top_paper_ref })}
+                />
+                <Button
+                  icon="tag"
+                  onClick={() => act("rename_paper", { ref: top_paper_ref })}
+                />
+                <Button
+                  icon="eject"
+                  onClick={() => act("remove_paper", { ref: top_paper_ref })}
+                />
+              </Flex.Item>
+            </Flex>
+            {paper.length > 0 && (
+              <>
```

Comment:
You should do here the same as with folders, only output a divider based on length, must the rest of the loop can be unwrapped to reduce the overall indentation amd nesting.